models/rubric.js

```diff
@@ -2,14 +2,40 @@
 module.exports = (sequelize, DataTypes) => {
   var Rubric = sequelize.define('Rubric', {
     name: {
-      allowNull: false,
+      allowNull: true,
       type: DataTypes.STRING,
     },
     iconName: {
       allowNull: true,
-      type: DataTypes.STRING
+      type: DataTypes.STRING,
+      defaultValue: 'work',
+    },
+    iconColor: {
+      allowNull: true,
+      type: DataTypes.STRING,
+      defaultValue: 'rose',
+    },
+    description: DataTypes.TEXT,
```

Comment:
Should it be possible for the description to have a null value? Or at least some sort of defaultValue? If a user doesn't enter anything for the description, what will happen?

Reply:
In that case, it will be null, which is okay and handled by the sequelize orm,